### PR TITLE
chore: Prepare for the initial release.

### PR DESCRIPTION
**Pull request by shane-tomlinson**

### CODE REVIEW

**Comment by shane-tomlinson** [comment]
@mozilla/fxa-devs - r?

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
-bower_components
 node_modules
+.vscode
```
Comment (philbooth) [inline .gitignore]: Don't we have a preference for people to use their local global ignore file for their own environment-related stuff, and limit the project ignore files for stuff that is generated by the project itself?

**Review by philbooth — APPROVED** [review]
LGTM!

File: .gitignore (hunk 1 above)
Comment (shane-tomlinson) [reply .gitignore]: TBH, I didn't know that was a thing until reading this comment!

https://stackoverflow.com/questions/7335420/global-git-ignore#7335487

**Pull request merged**